rest_framework/fields.py

```diff
@@ -873,20 +875,28 @@ def to_internal_value(self, data):
 
 # Number types...
 
-class IntegerField(Field):
+
+class NumberField(Field):
```

Comment:
That's right, yup. We *could* consider a new base class, but that's a separate issue.